File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/mr/cimrpatsigns/d/VitalSignsDTO.java
package iih.ci.mr.cimrpatsigns.d;

import xap.mw.core.data.*;
import xap.mw.coreitf.d.*;
import java.math.BigDecimal;

/**
 * 实体 DTO数据 
 * 
 */
public class VitalSignsDTO extends BaseDTO {
	private static final long serialVersionUID = 1L;
	
	/**
	 * 就诊id
	 * @return String
	 */
	public String getId_ent() {
		return ((String) getAttrVal("Id_ent"));
	}
	/**
	 * 就诊id
	 * @param Id_ent
	 */
	public void setId_ent(String Id_ent) {
		setAttrVal("Id_ent", Id_ent);
	}
	/**
	 * 测量时间
	 * @return FDateTime
	 */
	public FDateTime getDt_vt() {
		return ((FDateTime) getAttrVal("Dt_vt"));
	}
	/**
	 * 测量时间
	 * @param Dt_vt
	 */
	public void setDt_vt(FDateTime Dt_vt) {
		setAttrVal("Dt_vt", Dt_vt);
	}
	/**
	 * 体温
	 * @return String
	 */
	public String getTemperature() {
		return ((String) getAttrVal("Temperature"));
	}
	/**
	 * 体温
	 * @param Temperature
	 */
	public void setTemperature(String Temperature) {
		setAttrVal("Temperature", Temperature);
	}
	/**
	 * 脉搏
	 * @return String
	 */
	public String getPulse() {
		return ((String) getAttrVal("Pulse"));
	}
	/**
	 * 脉搏
	 * @param Pulse
	 */
	public void setPulse(String Pulse) {
		setAttrVal("Pulse", Pulse);
	}
	/**
	 * 呼吸
	 * @return String
	 */
	public String getBreathing() {
		return ((String) getAttrVal("Breathing"));
	}
	/**
	 * 呼吸
	 * @param Breathing
	 */
	public void setBreathing(String Breathing) {
		setAttrVal("Breathing", Breathing);
	}
	/**
	 * 收缩压
	 * @return String
	 */
	public String getSystolicbloodpressure() {
		return ((String) getAttrVal("Systolicbloodpressure"));
	}
	/**
	 * 收缩压
	 * @param Systolicbloodpressure
	 */
	public void setSystolicbloodpressure(String Systolicbloodpressure) {
		setAttrVal("Systolicbloodpressure", Systolicbloodpressure);
	}
	/**
	 * 舒张压
	 * @return String
	 */
	public String getDiastolicbloodpressure() {
		return ((String) getAttrVal("Diastolicbloodpressure"));
	}
	/**
	 * 舒张压
	 * @param Diastolicbloodpressure
	 */
	public void setDiastolicbloodpressure(String Diastolicbloodpressure) {
		setAttrVal("Diastolicbloodpressure", Diastolicbloodpressure);
	}
}

File: client/iih_stub/iih.ci_stub/iih.ci.mr/cimrvt/d/VitalSignsDTO.cs
using System;
using xap.mw.core.data;
using xap.mw.coreitf.d;

namespace iih.ci.mr.cimrpatsigns.d
{
    /// <summary>
    /// VitalSignsDTO 的摘要说明。
    /// </summary>
    public class VitalSignsDTO : BaseDTO {

        public VitalSignsDTO() {
 
        }

        /// <summary>
        /// 就诊id
        /// </summary>
		public string Id_ent {
            get { return getAttrVal<string>("Id_ent",null); }
            set { setAttrVal<string>("Id_ent", value); }
        }

        /// <summary>
        /// 测量时间
        /// </summary>
		public DateTime? Dt_vt {
            get { return getAttrVal<FDateTime>("Dt_vt",null); }
            set { setAttrVal<FDateTime>("Dt_vt", value); }
        }

        /// <summary>
        /// 体温
        /// </summary>
		public string Temperature {
            get { return getAttrVal<string>("Temperature",null); }
            set { setAttrVal<string>("Temperature", value); }
        }

        /// <summary>
        /// 脉搏
        /// </summary>
		public string Pulse {
            get { return getAttrVal<string>("Pulse",null); }
            set { setAttrVal<string>("Pulse", value); }
        }

        /// <summary>
        /// 呼吸
        /// </summary>
		public string Breathing {
            get { return getAttrVal<string>("Breathing",null); }
            set { setAttrVal<string>("Breathing", value); }
        }

        /// <summary>
        /// 收缩压
        /// </summary>
		public string Systolicbloodpressure {
            get { return getAttrVal<string>("Systolicbloodpressure",null); }
            set { setAttrVal<string>("Systolicbloodpressure", value); }
        }

        /// <summary>
        /// 舒张压
        /// </summary>
		public string Diastolicbloodpressure {
            get { return getAttrVal<string>("Diastolicbloodpressure",null); }
            set { setAttrVal<string>("Diastolicbloodpressure", value); }
        }
        /// <summary>
        /// 返回属性列表
        /// </summary>
        /// <returns></returns>
        public override string[] getAttrNames()
        {
            return new string[]{ "Id_ent", "Dt_vt", "Temperature", "Pulse", "Breathing", "Systolicbloodpressure", "Diastolicbloodpressure"};
        }
        
        /// <summary>
        /// 返回全路径DO类名
        /// </summary>
        /// <returns></returns>
        public override string getFullClassName()
        {
            return "iih.ci.mr.cimrpatsigns.d.VitalSignsDTO";
        }
    }
}
